Dockerfile

```diff
@@ -11,24 +11,25 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 #
-FROM openzipkin/zipkin:2.12.1
-MAINTAINER OpenZipkin "http://zipkin.io/"
+
+FROM alpine
+
+WORKDIR /zipkin-gcp
 
 ENV ZIPKIN_GCP_REPO https://jcenter.bintray.com
 ENV ZIPKIN_GCP_VERSION 0.10.1
-# Readback is currently not supported
-ENV QUERY_ENABLED false
-# must match JRE, in this case zipkin:2.12.1 uses 1.8.0_191
-# see https://github.com/square/okhttp/blob/master/pom.xml for example mappings
-# this and the download of alpn-boot will be removed with https://github.com/openzipkin/docker-jre-full/issues/9
-ENV ALPN_VERSION 8.1.13.v20181017
 
-RUN apk add unzip && \ 
-  curl -SL $ZIPKIN_GCP_REPO/org/mortbay/jetty/alpn/alpn-boot/$ALPN_VERSION/alpn-boot-$ALPN_VERSION.jar > alpn-boot.jar && \
+RUN apk add curl unzip && \
   curl -SL $ZIPKIN_GCP_REPO/io/zipkin/gcp/zipkin-autoconfigure-storage-stackdriver/$ZIPKIN_GCP_VERSION/zipkin-autoconfigure-storage-stackdriver-$ZIPKIN_GCP_VERSION-module.jar > stackdriver.jar && \
   echo > .stackdriver_profile && \
   unzip stackdriver.jar -d stackdriver && \
-  rm stackdriver.jar && \
-  apk del unzip
+  rm stackdriver.jar
+
+FROM openzipkin/zipkin:2.12.5
+MAINTAINER OpenZipkin "http://zipkin.io/"
+
+ENV ZIPKIN_GCP_VERSION 0.10.1
```

Comment:
nit this isn't used here as it is just used during the download

Reply:
I was a bit unsure about whether this is a sort of public API for images built on top of this - I have these sort of variables duplicated in all the dockerfiles so far. It they're not meant for users I'll delete them everywhere - WDYT?

Reply:
yep they were only meant for the download operations. there's an /info endpoint which has the zipkin version in it, but it we wanted to, we could consider adding the aws or gcp version, but it would likely be there and not in env.

Anyway it isn't critical, but yeah please do cleanup the others if you don't mind!